rank/scripts/createIndex.ts

```diff
@@ -0,0 +1,99 @@
+import { code, error, info, success } from './utils'
+
+import { getNamespaceFromIndexName } from '../types/searchTemplate'
+import { getRankClient } from '../services/elasticsearch'
+import { listIndices } from '../services/search-templates'
+import { parse } from 'path'
+import prompts from 'prompts'
+import { readdirSync } from 'fs'
+
+global.fetch = require('node-fetch')
+
+async function go() {
+  // Users should be able to create new indices by modifying mappings they've
+  // fetched from the rank cluster, where those source indices still exist.
+  const configFiles = readdirSync('./data/indices/')
+    .filter((fileName) => fileName.includes('.json'))
+    .map((fileName) => parse(fileName).name)
+  const sourceIndices = await listIndices()
+  const validIndices = sourceIndices.filter((value) =>
+    configFiles.includes(value)
+  )
+
+  const sourceIndex = await prompts({
+    type: 'select',
+    name: 'value',
+    message: 'Which index config are you working from?',
+    choices: validIndices.map((choice) => ({ title: choice, value: choice })),
```

Comment:
Convention is to name the mapped item as to what it is:
```suggestion
    choices: validIndices.map((index) => ({ title: index, value: index })),
```